core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -215,7 +213,7 @@ private void buildPhysicalDestinationName() {
 			}
 		}
 		builder.append(getUriPattern());
-		builder.append("; method: ").append(getMethods());
+		builder.append("; method: ").append(getAllMethods().stream().map(Enum::name).collect(Collectors.joining(",")));
```

Comment:
Dit doet toch hetzelfde als getMethods() voorheen? Ik vond dat leesbaarder in een losse method

Reply:
Mmm, nu je dit zo vraagt, bedenk ik me nu dat ik het wat schoner kan maken. Dat zal ik zo committen. 
De omzetting naar een aan elkaar geplakte String is verder nergens meer nodig behalve hier. En die wil ik ook niet aanbieden als `public`. 
Ik kan `.stream().map(Enum::name).collect(Collectors.joining(",")));` in een losse method `getMethodsAsString()` maken oid, maar vraag me af of dat veel mooier is per se.

Reply:
Helaas weer iets van de opschoning moeten terugdraaien, omdat `MapPropertyDescriptorsTest#testPropertyDescriptorsBeingRegistered` checkt of de setters en getters van hetzelfde type is. We willen expliciet op de setter `HttpMethod... method` hebben en de getter laten werken met de interne List<HttpMethod>, dus kan ik die niet dezelfde naam geven of `@Getter` gebruiken helaasch.

Reply:
Ja klopt, beanutils gaan er van uit dat de getters en setters het zelfde type hebben, de `MapPropertyDescriptorsTest` dwingt dit voor ons af zodat we geen rare situaties krijgen.